### PR TITLE
Consultation update page form UI updated

### LINKED ISSUE #9067: Update the form UI on the Consultation Update page
**Describe the bug**
Update the form UI on the Consultation Update page

**To Reproduce**

1. Go to 'patient button on sidebar'
2. Click on 'Click on active consultation'
3.  Switch to "Daily Rounds" 
4. Click on "View Details"
5. See error

**Expected behavior**
good ui

**Screenshots**
Image

**Desktop (please complete the following information):**

- OS: [windows]
- Browser [chrome]
- Version [130.0.6723.117]

### PR DESCRIPTION
**Pull request by modamaan**
## Proposed Changes

- Fixes #9067 
- Older UI
Screenshot 2024-11-10 070646

Updated UI
Screenshot 2024-11-10 070815

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Summary by CodeRabbit

- **New Features**
	- Updated the layout of the Daily Round List Details to a card-based design for improved visual representation.
	- Enhanced organization of health metrics and additional sections for respiratory information.

- **Bug Fixes**
	- Improved dynamic waiting for visibility in the Patient Log update process, replacing static wait times.
	- Added timeout handling for notification visibility checks in the verification command.

- **Documentation**
	- Updated method signatures to reflect changes in implementation.

### CODE REVIEW

**Review by rithviknishad — COMMENTED** [review]
I'd recommend to change to using this component (https://github.com/ohcnetwork/care_fe/blob/develop/src/components/LogUpdate/CriticalCarePreview.tsx) instead as that's the ones used for newer daily rounds and making improvements there. Do rename the component too when doing that to something generic (`LogUpdatePreviewPage` / `LogUpdateDetails`).

**Comment by modamaan** [comment]
Screenshot 2024-11-11 203615
@rithviknishad  Do you want to update this component?

**Comment by rithviknishad** [comment]
Yup

**Comment by rithviknishad** [comment]
Do delete the old component too once done

**Comment by modamaan** [comment]
@rithviknishad 
https://github.com/user-attachments/assets/8e856e82-a938-4c07-9867-1cc4834674f5
Screenshot 2024-11-12 114306

**Comment by nihal467** [comment]
@modamaan the build itself is failing, tag me for QA review once the PR is ready for review

**Comment by nihal467** [comment]
@modamaan what is the status of this PR

**Comment by nihal467** [comment]
@modamaan @rithviknishad the log update cypress are failing, fix the test and merge conflict

File: src/components/ui/badge.tsx
Comment (rithviknishad) [inline src/components/ui/badge.tsx:1]: why was this changed? i don't see this solving anything.

**Comment by rithviknishad** [comment]
Closing as this has been removed in the actively developed re-architecture branch in favour of Questionnaire.

https://github.com/ohcnetwork/care_fe/pull/9262

image

**Pull request closed without merging**